components/create_post/create_post.jsx

```diff
@@ -953,7 +953,12 @@ export default class CreatePost extends React.Component {
         let emojiPicker = null;
         if (this.props.enableEmojiPicker && !readOnlyChannel) {
             emojiPicker = (
-                <span className='emoji-picker__container'>
+                <span
+                    role='button'
+                    tabIndex='0'
+                    aria-label='Click to open emojies popup'
+                    className='emoji-picker__container'
```

Comment:
Can you add the same in RHS as well?. It would be in the `create_comment.jsx` file.